bambi/interpret/create_data.py

```diff
@@ -1,136 +1,131 @@
 import itertools
+
 from typing import Union
+from statistics import mode
 
 import numpy as np
 import pandas as pd
 
+from pandas.api.types import (
+    is_categorical_dtype,
+    is_float_dtype,
+    is_integer_dtype,
+    is_numeric_dtype,
+    is_object_dtype,
+    is_string_dtype,
+)
```

Comment:
I think many, if not all, of these functions will be deprecated. See for example https://github.com/pandas-dev/pandas/blob/2a953cf80b77e4348bf50ed724f8abc0d814d9dd/pandas/core/dtypes/common.py#L469-L511

However, when I tried to replace it with what was suggested in the warning, it didn't work. 

I'm not saying we should replace these helpers right now as it may require a broader refactor, I just flag it to keep it in mind.

Reply:
Thanks for the heads up!